tests/neg/i15855.scala

```diff
@@ -0,0 +1,10 @@
+// crash.scala
+import scala.language.implicitConversions
```

Comment:
Are the `implicitConversions` language imports required? (Here and in the other test)